target/linux/mpc85xx/files/arch/powerpc/boot/dts/msm460.dts

```diff
@@ -190,6 +197,13 @@
 				  0x1000000 0x0 0x0
 				  0x1000000 0x0 0x0
 				  0x0 0x100000>;
+
+			wifi@0,0 {
+				reg = <0x0000 0 0 0 0>;
+				nvmem-cells = <&macaddr_hwinfo_1f9bd 16>;
+				nvmem-cell-names = "mac-address";
+				qca,led-pin = /bits/ 8 <3>;
```

Comment:
Did you test this on your board? The radio is on a socketed card and although i know you have pins for RF state on the pcie pinout, i don't understand the benefit of this move.

Reply:
I did test the old version (userspace) and it's not working with 0x1f9bd (on one device). Below is a bid parser that I put together in 10 mins. ID 40 is the MAC address

[bid_parser.py.txt](https://github.com/user-attachments/files/17085088/bid_parser.py.txt)

Reply:
The benefit is that now the ath9k LEDs get created with a proper GPIO number. They're created anyway, so might as well create them properly.

Btw, the wdr4900v1 (also mpc85xx) is using this with no problems. I used qca,led-pin there as that's what the platform code was doing before I migrated it to dts.

Reply:
@neheb The WDR4900 might have the sink wired up to the GPIO and driving it from the ath9k chip. At least that's how i've seen it done sometimes. This is not platform-specific but depends on the electrical design

Reply:
right. This needs testing. My suspicion is that it works.

Reply:
Great. Removed wifi change.